etc/module.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="JustBetter_OptimizeFlatTables" setup_version="1.0.0">
+        <sequence>
+            <module name="Magento_Eav" />
```

Comment:
I see that `Magento\Catalog\Model\ResourceModel\Attribute` is also used. So wouldn't it be better to add `Magento_Catalog` here as well?
```suggestion
            <module name="Magento_Eav" />
            <module name="Magento_Catalog" />
```